src/content/capture/video-video/js/main.js

```diff
@@ -11,6 +11,12 @@ const leftVideo = document.getElementById('leftVideo');
 const rightVideo = document.getElementById('rightVideo');
 
 leftVideo.addEventListener('canplay', () => {
-  const stream = leftVideo.captureStream();
+  let stream;
+  const fps = 0;
+  if (leftVideo.mozCaptureStream) {
```

Comment:
This should preferably be swapped so in case both `captureStream` and `mozCaptureStream` are present, `captureStream` is chosen.

Thanks for fixing things for Firefox! (I am working towards eventually shipping the non-prefixed API but there's plenty left to do)

Reply:
Fixed